API.md

```diff
@@ -751,6 +751,17 @@ Defines the fit mode (default zoom level) of the viewer.
 />
 ```
 
+#### fitPolicy
+
+Defines the fit mode (default zoom level) of the viewer.
+0:fit both(default), 1:fit width 2:fit height
```

Comment:
use a table to explain the values so it is clearly presented to customers

Reply:
done

Reply:
well.... with the table you can remove line 758....

Reply:
and indicate default in your table

Reply:
done